experimental/packages/exporter-trace-otlp-grpc/test/OTLPTraceExporter.test.ts

```diff
@@ -49,20 +49,23 @@ const includeDirs = [
   path.resolve(__dirname, '../../otlp-grpc-exporter-base/protos'),
 ];
 
-const address = 'localhost:1501';
+const httpAddr = 'localhost:1501';
```

Comment:
Can we explicitly spell out the http/https protocol in the URL? The protocols are concatenated in the tests previously. However, the concatenation has been removed in this patch.

This partial address works because `validateAndNormalizeUrl` fills the missing protocol with `https`.

Reply:
Actually, `validateAndNormalizeUrl` removes protocol and path and only returns hostname and port. But your suggestion still makes sense.